src/main/scala/codacy/plugins/DockerTest.scala

```diff
@@ -12,9 +12,17 @@ case class Sources(mainSourcePath: Path, directoryPaths: Seq[Path])
 
 object DockerTest {
 
-  private val allTestKey = Seq("all")
   private val possibleTests = Seq(JsonTests, PluginsTests, PatternTests)
-  private val possibleTestNames = allTestKey ++ possibleTests.map(_.opt)
+  private val possibleTestsWithUda = SourceTests +: possibleTests
+
+  private val possibleTestNames = config.keySet
+
+  private lazy val config = Map(
+    "all" -> possibleTests,
+    "allWithUdas" -> possibleTestsWithUda
+  ) ++ possibleTestsWithUda.map{ case test =>
```

Comment:
I don't think it's clear to have the ++ possibleTestsWithUda.map{ ... here. But maybe I am missing something.